gemini/prompts/prompt_optimizer/vertex_ai_prompt_optimizer_ui.ipynb

```diff
@@ -0,0 +1,907 @@
+{
```

Comment:
<p>Can we have docstring for all helper functions? I think it is very important to help understanding the UI. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1099/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done.</p>